dangerzone/cli.py

```diff
@@ -33,6 +33,18 @@ def print_header(s: str) -> None:
     flag_value=True,
     help=f"Archives the unsafe version in a subdirectory named '{ARCHIVE_SUBDIR}'",
 )
+@click.option(
+    "--config-dir",
+    default=util.get_default_config_dir(),
+    show_default=True,
+    help="The user directory where Dangerzone will store configuration files",
+)
+@click.option(
+    "--cache-dir",
+    default=util.get_default_cache_dir(),
+    show_default=True,
+    help="The user directory where Dangerzone will store temporary files",
+)
```

Comment:
> Can you provide a user story where this may be needed?

To be fair, this is more of a dev thing. I agree that it's rare that regular users will ever need it.

Personally, I've come across situations in projects where it helped being able to change their config directory, so that you can test things out without affecting the original config. We basically do this in our QA tests. But this would apply to power users only.

> But if we want to provide a custom dir for these things, couldn't we theoretically also monkey-patch / mock in the tests the `appdirs.user_config_dir()` function and get the same result with less code?

Cool, if you feel that it complicates things, it's probably a good sign that the code needs to go. I'll send a version that works with monkey-patching.